assets/styles/_print.scss

```diff
@@ -0,0 +1,84 @@
+@media print {
+  *,
+  *:before,
+  *:after {
+    background: transparent !important;
+    color: #000 !important;
+    /* Black prints faster */
+    box-shadow: none !important;
+    text-shadow: none !important;
+  }
+
+  [is='flynt-navigation-main'],
+  [is='flynt-navigation-burger'],
+  [is='flynt-navigation-footer'],
+  [is="flynt-block-cookie-notice"],
+  .button,
+  footer {
+    display: none !important;
+  }
+
+  a {
+    text-decoration: underline;
+    font-weight: bold;
+  }
+
+  a[href]:after {
+    content: " (" attr(href) ")";
+  }
+
+  a[href^="/"]:after {
```

Comment:
I assume this would not show the urls of internal links, right? Why should we hide those?

Reply:
Was something I found when searching for print styles on the web. I removed it.